src/main/archetype/core/pom.xml

```diff
@@ -34,66 +33,57 @@
                 <artifactId>sling-maven-plugin</artifactId>
             </plugin>
             <plugin>
-                <groupId>org.apache.felix</groupId>
-                <artifactId>maven-bundle-plugin</artifactId>
-                <extensions>true</extensions>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-maven-plugin</artifactId>
                 <executions>
                     <execution>
-                        <id>generate-osgi-metadata-for-unittests</id>
+                        <id>bnd-process</id>
                         <goals>
-                            <goal>manifest</goal>
-                        </goals>
-                        <phase>process-classes</phase>
-                    </execution>
-                    <execution>
-                        <id>scr-metadata</id>
-                        <goals>
-                            <goal>manifest</goal>
+                            <goal>bnd-process</goal>
                         </goals>
                         <configuration>
-                            <supportIncrementalBuild>true</supportIncrementalBuild>
+                            <bnd><![CDATA[
+#set( $startbrace = "{" )
+#set( $endbrace = "}" )
+Bundle-Category: ${componentGroupName}
+-exportcontents: $${startbrace}packages;VERSIONED${endbrace}
+Sling-Model-Packages: ${package}.core.models
+-snapshot: $${startbrace}tstamp;yyyyMMddHHmmssSSS${endbrace}
+Bundle-DocURL:
+Import-Package: javax.inject;version="0.0.0",javax.annotation;version="0.0.0",*
```

Comment:
Although this is a compatible change, I think that the package imports should be automatically determined during build-time and not make all projects using this archetype import `javax.inject` and `javax.annotation`.

Reply:
@raducotescu I made a few tests: without the `Import-Package` statement the generated bundle is not started on previous AEM versions due to missing dependencies, whereas it is correctly started when this statement is available.

Reply:
I think that for the correct imports you should use the following dependencies in the `core` pom file:
```
<dependency>
    <groupId>org.apache.geronimo.specs</groupId>
    <artifactId>geronimo-atinject_1.0_spec</artifactId>
    <version>1.0</version>
</dependency>
<dependency>
    <groupId>org.apache.geronimo.specs</groupId>
    <artifactId>geronimo-annotation_1.3_spec</artifactId>
    <version>1.0</version>
</dependency>
```
instead of
```
<dependency>
    <groupId>javax.inject</groupId>
    <artifactId>javax.inject</artifactId>
</dependency>
<dependency>
    <groupId>javax.annotation</groupId>
    <artifactId>javax.annotation-api</artifactId>
</dependency>
```

Reply:
@raducotescu I redid some tests with the changes that you suggested (using the `geronimo` bundles and removing the `Import-Package` statement in the configuration of the `bnd` plugin): the generated bundle is not started due to missing dependencies.
I will merge this PR and create an improvement issue with the changes that you have suggested.

Reply:
I created a new issue at https://github.com/adobe/aem-project-archetype/issues/211